src/certificate.rs

```diff
@@ -166,11 +198,11 @@ impl Certificate {
     #[cfg(feature = "untested")]
     pub fn delete(yubikey: &mut YubiKey, slot: SlotId) -> Result<(), Error> {
         let txn = yubikey.begin_transaction()?;
-        write_certificate(&txn, slot, None, 0)
+        write_certificate(&txn, slot, None, CertInfo::Uncompressed)
     }
 
     /// Initialize a local certificate struct from the given bytebuffer
-    pub fn new(cert: impl Into<Buffer>) -> Result<Self, Error> {
+    pub fn from_bytes(cert: impl Into<Buffer>) -> Result<Self, Error> {
```

Comment:
This is cool, but perhaps also add a `impl<'a> TryFrom<&'a [u8]> for Certificate`?

Reply:
Done.